records_mover/url/s3/s3_file_url.py

```diff
@@ -55,6 +55,7 @@ def open(self, mode: str = "rb") -> IO[bytes]:
                            key_id=self.key,
                            mode=mode,
                            session=self._boto3_session)
+        # TODO: upgrade probably broke this - look it up
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker